Leave/leave_db.py

```diff
@@ -14,6 +14,14 @@ def GetLeavesByMemberID(member_id):
     leaves = [dict(zip([column[0] for column in db.GetDBCursor().description], row)) for row in db.GetDBCursor().fetchall()]
     return leaves
 
+def GetLeavesBetween(start_date, end_date, member):
+    if (member == None):
+        db.GetDBCursor().execute(f"SELECT * FROM [leaves] WHERE date >= '{start_date}' AND date <= '{end_date}'")
+    else:
+        db.GetDBCursor().execute(f"SELECT * FROM [leaves] WHERE date >= '{start_date}' AND date <= '{end_date}' AND member_id = {member.id}")
```

Comment:
probably want to separate declaring the query and the actual execution of the query.

Reply:
Done